package.json

```diff
@@ -28,17 +31,17 @@
     "assertion helpers"
   ],
   "devDependencies": {
-    "babel-cli": "6.5.1",
-    "babel-core": "6.5.1",
-    "babel-plugin-transform-object-rest-spread": "6.6.5",
-    "babel-preset-es2015": "6.1.18",
-    "chai": "3.5.0",
-    "eslint": "3.3.1",
-    "eslint-config-airbnb": "2.1.1",
-    "mocha": "3.0.2",
-    "proxyquire": "1.7.10",
-    "sinon": "1.17.5",
-    "sinon-chai": "2.8.0",
+    "babel-cli": "~6.5.1",
```

Comment:
I thought using `~` would be ideal here, so we'd know what versions originally worked, in case of an unexpected breaking change, but also not have to update this file every time a minor release comes out.